src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs

```diff
@@ -258,19 +258,17 @@ public void CopyRuntimeSettingsTo(JsonPropertyInfo other)
             return (TAttribute)PropertyInfo?.GetCustomAttribute(typeof(TAttribute), inherit: false);
         }
 
-        public abstract void ApplyNullValue(JsonSerializerOptions options, ref ReadStack state);
-
         public abstract IList CreateConverterList();
 
-        public abstract Type GetConcreteType(Type interfaceType);
+        public abstract Type GetDictionaryConcreteType();
 
         public abstract void Read(JsonTokenType tokenType, JsonSerializerOptions options, ref ReadStack state, ref Utf8JsonReader reader);
         public abstract void ReadEnumerable(JsonTokenType tokenType, JsonSerializerOptions options, ref ReadStack state, ref Utf8JsonReader reader);
         public abstract void SetValueAsObject(object obj, object value);
 
         public abstract void Write(JsonSerializerOptions options, ref WriteStackFrame current, Utf8JsonWriter writer);
 
-        public abstract void WriteDictionary(JsonSerializerOptions options, ref WriteStackFrame current, Utf8JsonWriter writer);
+        public virtual void WriteDictionary(JsonSerializerOptions options, ref WriteStackFrame current, Utf8JsonWriter writer) { }
```

Comment:
What motivated this change?

Reply:
To remove the method that was implemented in JsonPropertyInfoNullable - it wasn't needed since a Dictionary can't be nullable.